readthedocs/core/symlink.py

```diff
@@ -144,26 +144,26 @@ def symlink_cnames(self, domain=None):
         if domain:
             domains = [domain]
         else:
-            domains = Domain.objects.filter(project=self.project)
+            domains = (dom.domain for dom in Domain.objects.filter(project=self.project))
```

Comment:
Why did you convert this query into a tuple?

In case you want to go in that direction, you should use `.values_list('domain', flat=True)`

Reply:
@humitos 
My motive was to use a `generator expression` there.

Reply:
I have changed it to list comprehension..
Actually, the function requires the list of strings, that is why list comprehension.

Reply:
Thanks for guiding me.
I have implemented the above method as told by you.